src/main/java/net/fabricmc/meta/web/models/BaseVersion.java

```diff
@@ -22,12 +22,19 @@
 
 	String version;
 	boolean stable = false;
+	String semver;
```

Comment:
Create a child class of this.

Reply:
What do you mean?

Reply:
Make a child class of BaseVersion that has this field. Prob call it something like MinecraftVersion